docs/operations/export_api.md

```diff
@@ -217,12 +217,30 @@ The export server certificate is valid for 7 days after which it is rotated by d
 #### External link certificates
 The external link certificates are associated with the Ingress/Route that points to the service created by the KubeVirt operator. The CA that signed the Ingress/Route will part of the certificates. 
 
+### TTL (Time to live) for an Export
+For various reasons (security being one), users should be able to specify a TTL for the VMExport objects that limits the lifetime of an export.  
+This is done via the `ttlDuration` field which accepts a k8s [duration](https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#Duration),  
+which defaults to 2 hours when not specified.
+```yaml
+apiVersion: export.kubevirt.io/v1alpha1
+kind: VirtualMachineExport
+metadata:
+    name: example-export
+spec:
+    source:
+        apiGroup: "kubevirt.io"
+        kind: VirtualMachine
+        name: example-vm
+    tokenSecretRef: example-token
+    ttlDuration: 1h
+```
+
 ### virtctl integration: vmexport
 The virtctl `vmexport` command allows users to interact with the export API in an easy-to-use way.
 
 `vmexport` uses two mandatory arguments:
-1. The vmexport **functions** (create|delete|download).
-2. The VirtualMachineExport **name**.
+* The vmexport **functions** (create|delete|download).
```

Comment:
```suggestion

* The vmexport **functions** (create|delete|download).
```

There has to be an empty line above the list for it to work in the parser.

Reply:
done! thx